src/clients/arango.ts

```diff
@@ -31,7 +30,7 @@ export class ArangoDBService {
     } catch (error) {
       LoggerService.error(
         'Error while executing query from arango with message:',
-        error,
+        error as any,
```

Comment:
instead of casting error as any, and making eslint sad, you can just change the logger class to allow Unknown, then do a type check on the  Error object. Can see sample https://github.com/ActioFRM/rule-003/blob/cfe5daca5e0b9cf251f4836b18bda99839d5d20e/src/services/logger.service.ts

Reply:
Done ✅